package.json

```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "json-stable-stringify": "^1.0.1",
     "lodash.kebabcase": "^4.1.1",
-    "nock": "^9.0.13",
+    "nock": "^11.1.0",
```

Comment:
Any reason why we did not upgrade to the latest version (v13)?

Reply:
The reason was that `@types/nock` doesn't go past this version. Though, as you pointed out, it's not needed anymore. I'll update the PR.